crates/sui-core/src/consensus_manager/mod.rs

```diff
@@ -68,31 +76,238 @@ impl ConsensusManager {
             parameters: consensus_config.narwhal_config().to_owned(),
             registry_service: registry_service.clone(),
         };
-
-        let metrics = ConsensusManagerMetrics::new(&registry_service.default_registry());
-
         Self::Narwhal(NarwhalManager::new(narwhal_config, metrics))
     }
 
+    /// Creates a new mysticeti manager enum.
```

Comment:
nit, I'd remove the word "enum" from this comment? It was confusing to me because it made it sound like you are *only* creating the enum, but actually you are also creating the manager itself here